client/systems.go

```diff
@@ -138,3 +139,48 @@ func (client *Client) RebootToSystem(systemLabel, mode string) error {
 	}
 	return nil
 }
+
+type InstallStep string
+
+const (
+	InstallStepSetupStorageEncryption InstallStep = "setup-storage-encryption"
+	InstallStepFinish                 InstallStep = "finish"
+)
+
+type SystemActionInstall struct {
```

Comment:
any reason for this to be exported?

Reply:
It's used in daemon for the argument unmarshal, hope that is okay.

Reply:
we are not very consistent in this area (with this kind of naming we usually would also have an Action field in this, SystemAction is the outlier but we don't have enough a lot of samples) but if we are sharing something then I think we should add json tags to InstallSystemOptions and tweak the fields and share that instead